gnosis/safe/safe_tx.py

```diff
@@ -415,22 +415,19 @@ def sign(self, private_key: str) -> bytes:
 
         # Insert signature sorted
         if account.address not in self.signers:
-            new_owners = self.signers + [account.address]
-            new_owner_pos = sorted(new_owners, key=lambda x: int(x, 16)).index(
-                account.address
-            )
-            self.signatures = (
-                self.signatures[: 65 * new_owner_pos]
-                + signature
-                + self.signatures[65 * new_owner_pos :]
+            new_tx_signatures = SafeSignature.parse_signature(
```

Comment:
Maybe just `signatures`, because `new` is not accurated to the variable, I mean the variable contains old signatures and new signatures. 
Other option could be unsorted_signatures is you want to remark that are unsorted and needs to be sorted.

Reply:
Updated here https://github.com/safe-global/safe-eth-py/pull/1060/commits/5c8cf1e79ba47af31cbe56a4ba54c60c69eca9a4